src/controller/home/homeController.ts

```diff
@@ -0,0 +1,56 @@
+import * as express from "express";
+import {Controller} from "./../controller";
+import {WeatherServiceInterface} from "./../../../src/library/services/weather/weatherServiceInterface";
+
+export class HomeController extends Controller {
+
+    private forecastService: WeatherServiceInterface;
+
+    constructor(forecastService: WeatherServiceInterface) {
+        super();
+        this.forecastService = forecastService;
+    }
+
+    public index(req: express.Request, res: express.Response) {
+
+        try {
+
+            res.render(
+                "default/index", {
+                    title: "ETaskr Test"
+                }
+            );
+
+        } catch (e) {
+            console.log(e);
```

Comment:
You wouldn't want to console.log on a prod env.

In PHP we usually have a DI'ed logger to handle error logging - PSR/LoggerInterface. Pretty sure Node would have the same just need to investigate.